### PR TITLE
Camera Rotation

### LINKED ISSUE #76: Implement Camera Rotation when User rotates Device (iPad)
When using the WeScan Framework on an iPad, the Camera View is not being rotated when the User rotates the Device. (eg. from Portrait to Landscape Mode)

### PR DESCRIPTION
**Pull request by JulianKahnert**
In 7c79c6f0c05f010a1b1af8cd4c86b330dc497b84 I have already fixed the missing "change of orientation" in the ScanViewController. However, the captured image is always displayed in portrait mode in the EditViewController. This is due to a wrong UIImage.imageOrientation.

I'm not sure where to set the imageOrientation of the newly captured image. Or if it should be changed at all, since this could lead the user to change to portrait mode in the EditViewController.
Does anyone have a hint for me?

fixes #76

### CODE REVIEW

File: WeScan/Scan/ScannerViewController.swift [hunk 1]
```diff
@@ -104,11 +104,14 @@ final class ScannerViewController: UIViewController {
         navigationController?.navigationBar.barStyle = .blackTranslucent
     }
     
+    override func viewWillLayoutSubviews() {
+        videoPreviewLayer.connection?.videoOrientation = AVCaptureVideoOrientation(rawValue: UIApplication.shared.statusBarOrientation.rawValue) ?? .portrait
+        videoPreviewLayer.frame = view.layer.bounds
+    }
```
Comment (AvdLee) [inline WeScan/Scan/ScannerViewController.swift]: Using this method is quite heavy. Also, it's not calling super here. Can we think of a better way to solve this?

Comment (mihailsalari) [reply WeScan/Scan/ScannerViewController.swift]: When the Batch scanning will be done? After I can fix that.

**Comment by JulianKahnert** [comment]
@AvdLee you are right. I will close this PR.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by willm132** [comment]
Has anyone been able to fix this yet? My app only works in landscape and the camera pops up in orientation.

**Pull request closed without merging**